frontend/src/models/devices.ts

```diff
@@ -136,6 +146,45 @@ export default createModel({
         return { devices: [], total: 0, error }
       }
     },
+    async fetchLogs({ id, from, maxDate }: any, globalState: any) {
```

Comment:
I think we should be getting logs independent from device list. Please reference the accounts.ts or applicationTypes.ts models and let's make a logs.ts model to handle the querying and data.